adapters/ucfunnel/ucfunnel_test.go

```diff
@@ -13,29 +13,13 @@ func TestJsonSamples(t *testing.T) {
 	adapterstest.RunJSONBidderTest(t, "ucfunneltest", NewUcfunnelBidder(nil, "http://127.0.0.1:8081/tmp12.json"))
 }
 
-func TestAddHeadersToRequest(t *testing.T) {
-	header := AddHeadersToRequest()
-
-	if header == nil {
-		t.Errorf("actual = %v expected != %v", nil, nil)
-	}
-
-	if header.Get("Content-Type") != "application/json;charset=utf-8" {
-		t.Errorf("actual = %s expected != %v", header.Get("Content-Type"), nil)
-	}
-
-	if header.Get("Accept") != "application/json" {
-		t.Errorf("actual = %s expected != %v", header.Get("Accept"), nil)
-	}
-}
-
 func TestUcfunnelAdapterNames(t *testing.T) {
-	adapter := NewUcfunnelAdapter(adapters.DefaultHTTPAdapterConfig, "http://localhost/bid")
+	adapter := NewUcfunnelBidder(nil, "http://localhost/bid")
 	adapterstest.VerifyStringValue(adapter.Name(), "ucfunnel", t)
 }
 
 func TestSkipNoCookies(t *testing.T) {
-	adapter := NewUcfunnelAdapter(adapters.DefaultHTTPAdapterConfig, "http://localhost/bid")
+	adapter := NewUcfunnelBidder(nil, "http://localhost/bid")
 	status := adapter.SkipNoCookies()
```

Comment:
We seem to don't need the first argument of `NewUcfunnelBidder` anymore, do we? This was also broght up in the `adapters/ucfunnel/ucfunnel.go` review